### DISCUSSION TITLE
Import from Recipe Keeper

### DISCUSSION THREAD

**Discussion by atrain2009** [mealie-recipes/mealie#3331] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Being able to import from another source - Recipe Keeper

### Please provide a concise description of the feature that would resolve your issue.

I'd love to be able to import my recipes from Recipe Keeper! I currently have over 900 recipes saved, but I'd love to be able to migrate to something self-hosted and open source. It looks like it exports as one big HTML file with an images folder for all of the pictures.

### Please consider and list out some caveats or tradeoffs made in your design decision

I don't know how many people use Recipe Keeper or what goes into adding a new import source.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by woodshop80** [comment]
I use Recipe Keeper also and would like to migrate to Mealie. It appears that you could import into Recipe Keeper from Mealie via a .zip file. Unfortunately I'm an end-user with little programming experience and have not had a chance to look at the import/export files and wouldn't know where to begin.

**Comment by derpferd** [comment]
FYI, created a PR implementing Recipe Keeper importer: #3642

**Reply by atrain2009** [reply]
Amazing, thank you so much!

**Comment by boc-the-git** [comment]
@atrain2009 @woodshop80 , @derpferd's PR is currently being merged and should be available on the nightly image in ~20 minutes time :)

**Reply by woodshop80** [reply]
Awesome!  Thank you!
